tests/integration/utils.py

```diff
@@ -93,6 +93,35 @@ def wait_for_complete_request(self, response):
 
         return response
 
+    def fetch_all_requests(self, request_json=None):
+        """
+        Fetch a list of requests from the Cachito API.
+
+        :param dict request_json: Request parameters and values (page, per_page, status, verbose)
+        :return: Object that contains response from the Cachito API
+        :rtype: list
+        """
+        if not request_json:
+            request_json = {}
+        result_url = f"{self._cachito_api_url}/requests"
```

Comment:
Optional: I prefer this name, but you can keep it the other way if you prefer.
```suggestion
        request_url = f"{self._cachito_api_url}/requests"
```

Reply:
Done, thanks!